src/common/telemetry/exe/main.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <syslog.h>
+
+#include <telemetry.hpp>
+
+void log_error(const std::string& message) {
+    syslog(LOG_ERR, "%s", message.c_str());
+}
+
+void log_info(const std::string& message) {
+    syslog(LOG_INFO, "%s", message.c_str());
+    std::cout << message << std::endl;
+}
+
+bool file_exists(const std::string& filepath) {
+    std::ifstream file(filepath.c_str());
+    return file.good();
+}
```

Comment:
This is not a deamon, it should not force logging to syslog.

Instead, use our logging library that entire OSConfig project uses, that logging automatically gets routed to syslog when the code runs as service.